core-nlp-example-master/target/maven-archiver/pom.properties

```diff
@@ -1,5 +1,5 @@
 #Generated by Maven
-#Mon Apr 29 16:14:38 IST 2019
+#Sat May 11 17:10:18 IST 2019
 version=1.0-SNAPSHOT
-groupId=nlp
-artifactId=exampl-pos
+groupId=com.techprimers.nlp
+artifactId=core-nlp-example
```

Comment:
Hey , I've changed that !

Reply:
Cool. Can you bring back the CoreNlpExample.java? I will merge this once that file is added back